drivers/iio/dac/ti-dac5571.c

```diff
@@ -328,7 +329,15 @@ static int dac5571_probe(struct i2c_client *client,
 	indio_dev->modes = INDIO_DIRECT_MODE;
 	indio_dev->channels = dac5571_channels;
 
-	spec = &dac5571_spec[id->driver_data];
+	if (dev->of_node){
```

Comment:
The kernel coding style requires a space before {. The } before the else should be on the same line as the else, with a space between the two. But in any case the curly braces should be removed, as the kernel coding style doesn't use them for single-line statements.

I recommend getting familiar with scripts/checkpatch.pl, it's a script that takes a patch file (which you can generate with `git format-patchl`) and checks for lots of common issues (covering both coding style violations and functional issues).